roles/ovb/bmc/tasks/packages.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Install the necessary packages to setup the bmc node
+
+- name: Install repos
+  yum:
+      name="{{ item }}"
+      state=present
+  with_items:
+      - 'https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm'
```

Comment:
just an idea, why not move this list to a variable under defaults/ and have the option for a user to override it if supplied.
Example: (under defaults/)
```
rpm_list:
    - 'https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm'
    - 'https://rdo.fedorapeople.org/rdo-release.rpm'
rpms: "{{ provisioner.plugins.bms.rpms | default(rpm_list) }}"
```